### PR TITLE
Monitoring features

### PR DESCRIPTION
**Pull request by Skydev0h**
* Grafana security hardening in caddy
* Updated transaction counter with new values
* Added supabase telemetry exporter (NB: The key is public, any user can get it from app)
* Fix gosh-mon tests to work with new login flow, also fixed some validation bugs, removed and disabled unneeded functionality
* Updated grafana dashboards to actual version
* Added code hash counting to tx-cnt-stat

### CODE REVIEW

**Review by samo3l — DISMISSED** [review]

File: .ci/Jenkinsfile.gosh-prepare-rc [hunk 1]
```diff
@@ -223,22 +224,18 @@ pipeline {
...
-                            }
-                        }
+                        sh """
+                            docker run --rm \
+                                -e GM_MODE='app-setup' \
+                                -e RUN_NOW=1 \
+                                -e CONFIG_SEED="\$(cat ../../tests/user.seed)" \
+                                -e CONFIG_APPURL=https://${BUCKET_NAME}.gosh.sh/ \
+                                -e CONFIG_REDIS_HOST="" \
+                                -e USERNAME=user1 \
```
Comment (Skydev0h) [inline .ci/Jenkinsfile.gosh-prepare-rc:234]: Must be `CONFIG_USERNAME`

File: .ci/Jenkinsfile.gosh-prepare-rc [hunk 2]
```diff
@@ -223,22 +224,18 @@ pipeline {
...
-                                        -e CONFIG_REDIS_HOST="" \
-                                        -v \$(pwd)/config:/home/node/app/config:ro \
-                                        -v \$(pwd)/test-errors:/home/node/app/errors \
-                                        gosh-mon
-                                """
-                            }
-                        }
+                        sh """
+                            docker run --rm \
+                                -e GM_MODE='app-setup' \
```
Comment (Skydev0h) [inline .ci/Jenkinsfile.gosh-prepare-rc:229]: The problematic test `root-check` was removed from `app.yml`, maybe better return all tests in order?

File: monitoring/caddy/caddy/Caddyfile [hunk 3]
```diff
@@ -67,17 +86,17 @@ https://pushgateway.monitoring.gosh.sh {
 }
 
 https://next-app.monitoring.gosh.sh {
-    reverse_proxy http://gooosh.matusevich.pro {
-        header_up Host gooosh.matusevich.pro
+    reverse_proxy http://gosh.matusevich.pro {
```
Comment (andrew-kurochkin) [inline monitoring/caddy/caddy/Caddyfile:89]: It should be passed in env variables to make it generic enough for tests

Comment (andrew-kurochkin) [reply monitoring/caddy/caddy/Caddyfile:89]: https://app.asana.com/0/1202815812933539/1203788451585095/f

**Review by andrew-kurochkin — APPROVED** [review]

File: monitoring/caddy/caddy/Caddyfile (hunk 3 above)
Comment (Skydev0h) [reply monitoring/caddy/caddy/Caddyfile:89]: This is not used anymore, will be removed in that task.

**Pull request merged**